src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -11,18 +11,18 @@
 import seedu.address.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a candidate identified using it's displayed index from the address book.
  */
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
+            + ": Deletes the candidate identified by the index number used in the displayed candidates list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted candidate: %1$s";
```

Comment:
Hey leon! Kind of being nitpicky but don't want to be nitpicky but HAHA. Could probably do `Deleted Candidate` in your next code related PR.

Reply:
Sure thing no problem~

With regards to your question on `%1$d` - tbh I have no idea at first, just copied and pasted from the original. Some quick google search gave a pretty good explanation:

<img width="817" alt="Screenshot 2022-03-13 at 12 03 27" src="https://user-images.githubusercontent.com/66457036/158044595-d70c35c8-bd6d-46c9-98dd-edc735468769.png">

Reply:
Right! Totally forgot about the formatting for `printf`. Yeah, I was wondering how it worked when it was a String. Guess that clarifies. Thanks @lzan98!!